pages/jotform-101.vue

```diff
@@ -49,23 +61,25 @@
 </template>
 
 <script>
-import KeyFeaturesOfJotform from '../components/jotform101/key-features-of-jotform.vue'
-import WhatMakesJotformStandOut from '../components/jotform101/what-makes-jotform-stand-out.vue'
-import JotformPopularUseCase from '../components/jotform101/popular-use-case.vue'
-import JotformLimitations from '../components/jotform101/limitations.vue'
+import JotformKeyFeatures from '../components/competitors/KeyFeatures.vue'
```

Comment:
Why not just name it `KeyFeatures`?

Reply:
I was thinking as we are using it on the jotform page let's name it JotformKeyFeatures. 
Yes you are right naming it only KeyFeatures would be appropriate here.